frontend/components/dashboard/CollateralTable.vue

```diff
@@ -60,23 +74,6 @@
                 %
             </span>
         </div>
-        <div slot="icon" slot-scope="record" class="Element">
-            <CurrencyIcon :currency-symbol="record.symbol" />
-        </div>
```

Comment:
Why did you need to remove `CurrencyIcon` and move `token address` column around to fix the problem? Is it unrelated improvement or a leftover?

Reply:
Alright! Did it also change the order of columns in the UI or had no effect on it?